homeassistant/components/media_player/sonos.py

```diff
@@ -71,6 +71,9 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     import soco
     global DEVICES
 
+    if hasattr(hass.config, 'host'):
```

Comment:
This is wrong. Every attribute on config will always need a default value. See `core.Config`.